[convai] add silence pricing disclaimer (#1091)
elevenlabs/elevenlabs-docs: fern/conversational-ai/pages/overview.mdx

  future.
</Note>

### Pricing during silent periods

> When a conversation is silent for longer than ten seconds, we reduce the inference of our turn-taking model and speech-to-text services until voice activity is detected again. This optimization means that extended periods of silence are charged at 5% of the usual per-minute cost.

Reviewer: ```suggestion
When a conversation is silent for longer than ten seconds, ElevenLabs reduces the inference of the turn-taking model and speech-to-text services until voice activity is detected again. This optimization means that extended periods of silence are charged at 5% of the usual per-minute cost.
```

> We are excited to see the kinds of applications this enables, for example long duration browsing assistants that are able to reactivate quickly when the user starts speaking again.

Reviewer: This isn't the tone we're going for.

```suggestion

```
